### ISSUE TITLE
Implement schedules for days / times?

### ISSUE THREAD

**Issue by Edison517** [OpenEVSE/ESP8266_WiFi_v2.x#224]
I apologize if this request is being worked on already, but if so, I couldn't find it.

I'd like the ability to set schedules for charging by day, and time, multiple times per day (if possible).

I have time-of-use billing and I would rather have the schedule of cheap rates (mostly at night) done on my OpenEVSE rather than my 2014 Volt. I know that right now I can set a daily schedule, but that doesn't help me when my weekend rates are the same as my nightly rates during the week :)

**Comment by glynhudson** [comment]
Yes, being able to set a weekly charging schedule is something we will be working on implementing in the near future.

**Comment by glynhudson** [comment]
![Screenshot 2019-04-30 at 12 39 15](https://user-images.githubusercontent.com/758844/56959475-6735b800-6b45-11e9-9824-ae5a5bd792c5.png)

Possible schedule UI 

https://github.com/openenergymonitor/ESP8266_Relay_Board/blob/master/firmware/html/control/thermostat.js
https://github.com/openenergymonitor/ESP8266_Relay_Board/blob/master/firmware/html/control/thermostat.html

Maybe keep the bars to visualise each day or the week but set the on/off time via a browser time picker. Android automatically then uses the nice inbuilt time picker
